shop/src/App.css

```diff
@@ -1,38 +1,77 @@
-.App {
-  text-align: center;
-}
+@media screen and (min-width: 1200px) {
+  .cardsContainer {
+    display: grid;
+    grid-template-columns: repeat(3, 1fr);
+    gap: 20px;
+    grid-auto-rows: maxmax(200px, auto);
```

Comment:
what is `maxmax` ?  i dont think I heard of it before

Reply:
It is minmax. not maxmax. 
OMG, my style.